entrypoint.sh

```diff
@@ -11,28 +11,19 @@ REGISTRATION_TOKEN=$(curl -s -X POST \
 
 UNIQUE_ID=$(uuidgen)
 
-# Register the runner
+# Register the runner:
+# - disable updates since we manage them manually via the container image
+#   - https://docs.github.com/en/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners#controlling-runner-software-updates-on-self-hosted-runners
+# - register as an ephemeral runner
+#   - https://docs.github.com/en/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners#using-ephemeral-runners-for-autoscaling
 ./config.sh \
       --unattended \
       --url "https://github.com/${REPO_OWNER}/${REPO_NAME}" \
       --token "${REGISTRATION_TOKEN}" \
       --name "${UNIQUE_ID}" \
       --work ../work-dir \
-      --replace
+      --replace \
+      --disableupdate \
+      --ephemeral
 
-cleanup() {
```

Comment:
Does the `--ephemeral` flag replace the need for this old cleanup code, or was there something else that deprecated this?

Reply:
Yep! 

> Github is deprecating the --once flag for the run script and [replacing it with the --ephemeral flag for the configure script](https://docs.github.com/en/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners#using-ephemeral-runners-for-autoscaling). Runners that are configured as ephemeral are automatically de-registered by GitHub, so we can safely remove the cleanup step from our entrypoint script (I also didn't think it was necessary to clean up any folders upon exit since the whole container is torn down at this point)